kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx

```diff
@@ -70,6 +70,11 @@ const ActionsCell: React.FC<CellContext<Topic, unknown>> = ({ row }) => {
           </>
         }
         danger
+        permission={{
+          resource: ResourceType.TOPIC,
+          action: [Action.VIEW, Action.CREATE, Action.DELETE],
+          value: name,
+        }}
       >
         Recreate Topic
       </DropdownItem>
```

Comment:
shouldn't this component be converted to `ActionDropdownItem` , cause `DropdownItem` component does not have that `props`.

Reply:
oh..i got it
and then i replaced DropdownItem to ActionDropdownItem. 
and i add disabled component.
Can i upload it?

<img width="507" alt="스크린샷 2023-03-10 오전 8 52 48" src="https://user-images.githubusercontent.com/7091756/224187661-b102107c-5946-40fd-997f-74cf7bd6a8e9.png">

Reply:
@Mgrdich can i review it?